pkgs/development/python-modules/mozilla-django-oidc/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, setuptools
+, django
+, josepy
+, requests
+, cryptography
+}:
+
+buildPythonPackage rec {
+  pname = "mozilla-django-oidc";
+  version = "4.0.1";
+  pyproject = true;
+
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner = "mozilla";
+    repo = "mozilla-django-oidc";
+    rev = "${version}";
+    hash = "sha256-72F1aLLIId+YClTrpOz3bL8LSq6ZhZjjtv8V/GJGkqs=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  dependencies = [
+    django
+    josepy
+    requests
+    cryptography
+  ];
+
+  pythonImportsCheck = [
```

Comment:
What do you mean by proper tests? It's one of my first contributions (I've got a few PR's open but not a single one has been merged or really reviewed yet). I read the [python language framework docs](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck-using-pythonimportscheck) and checked other modules, and as far as I can tell, thats what most modules are doing.

Reply:
I will try, I avoided it because I think it's a lot of work and I never did something like this before. I guess I also have to package https://github.com/peritus/bumpversion then because it's a requirement for the tests, right? (Currently trying if it also works without them...). I'd like to avoid packageing a 5 year old thing that is not longer supported and has been replace, but seams to be required still for these tests.

Reply:
I created these packages, but it seams like the tests cannot be imported. I'm going to create my own local module for peering-manager for now and maintain it myself.